plugins/rebalance/api/resp.go

```diff
@@ -0,0 +1,68 @@
+package rebalance
+
+import "github.com/pborman/uuid"
+
+// Status represents Rebalance Status
+type Status uint64
+
+const (
+	// GfDefragStatusNotStarted should be set only for a volume in which rebalance process is not started
+	GfDefragStatusNotStarted Status = iota
+	// GfDefragStatusStarted should be set only for a volume that has been just started rebalance process
+	GfDefragStatusStarted
+	// GfDefragStatusStopped should be set only for a volume that has been just stopped rebalance process
+	GfDefragStatusStopped
+	// GfDefragStatusComplete should be set only for a volume that the rebalance process is completed
+	GfDefragStatusComplete
+	// GfDefragStatusFailed should be set only for a volume that are failed to run rebalance process
```

Comment:
This comment is incorrect. The status will be failed if the rebalance process has failed on a node.